ocaml/libs/vhd/vhd_format/f.ml

```diff
@@ -2633,7 +2637,7 @@ functor
       | false, None ->
           false
 
-    let rec coalesce_request acc s =
+    let rec coalesce_request ?(coalesced_sectors = 1L) acc s =
```

Comment:
The new parameter could be explained in a comment. It is not obvious to me, especially because it it optional.

Reply:
done